src/pm/worktree/manager.py

```
@@ -0,0 +1,135 @@
+                capture_output=True,
+                text=True,
+            )
+            if result.returncode != 0:
+                raise RuntimeError(f"Failed to create worktree: {result.stderr.strip()}")
+
+        logger.info(f"Created worktree at {worktree_path} on branch {branch_name}")
+        return str(worktree_path)
+
+    def remove_worktree(self, worktree_path: str, repo_path: str) -> None:
+        """Remove a git worktree."""
+        result = subprocess.run(
```

Ruff (0.15.7)

[error] `subprocess` call: check for execution of untrusted input

(S603)